libraries/abstractions/common_io/include/iot_uart.h

```diff
@@ -116,23 +117,32 @@ typedef struct
 } IotUARTConfig_t;
 
 /**
- * @brief Initializes the UART peripheral of the board with default configuration.
+ * @brief Initializes the UART peripheral of the board.
```

Comment:
For IotUARTConfig_t data structure above. I think we need to make the variables more optimized and update the documentation. 

1) ulParity can be 8 bit -> ucParity, and update the documentation to be "0-> no parity,  1-> even parity and 2->odd_parity"

2) ulWordLength -> It can be ucWordLength as the max word length cannot be more than 4095 bits anyway.

3) ulStopbits -> It can be uint8_t -> ucStopbits as the max bits cannot be more than 4095. Also clarify 0 means no stop bits, 1 means 1 stop bit etc..

4) ulFlowControl -> change to uint8_t -> ucFlowControl -> 0 means disabled, 1 means enabled.

Reply:
Will update.

Reply:
looks good.